package.json

```diff
@@ -24,7 +24,6 @@
     "dotenv": "^8.2.0",
     "gh-pages": "^3.1.0",
     "memoize-one": "^5.1.1",
-    "node-sass": "^4.14.1",
```

Comment:
Noting that [`node-sass`](https://www.npmjs.com/package/node-sass) is deprecated. As long as the rest of the codebase is not using it!